.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Short description explaining the high-level reason for the new issue.
+
+## Current behavior
+
+-
+
+## Expected behavior
+
+-
+
+## Steps to replicate behavior
+
+1.
+
+
+## Screenshots
+
+__Current__
+
+
+__Expected__
```

Comment:
Would it be confusing for people reporting issues to provide expected screenshots?